gem/spec/fixtures/carthage_after/Example.xcodeproj/project.pbxproj

```diff
@@ -645,49 +366,43 @@
 			};
 			name = Release;
 		};
-		F98C3BA81C65354100389099 /* Debug */ = {
+		F9BC1C9B1CEFB77F00FE2BB8 /* Debug */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
-				INFOPLIST_FILE = ExampleUITests/Info.plist;
+				BUNDLE_LOADER = "$(TEST_HOST)";
+				CLANG_ANALYZER_NONNULL = YES;
+				FRAMEWORK_SEARCH_PATHS = "$(SRCROOT)/Carthage/Build/iOS";
+				HEADER_SEARCH_PATHS = "$(SRCROOT)/Carthage/Build/iOS/**";
+				INFOPLIST_FILE = AutoEarlGrey/Info.plist;
+				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = com.example.ExampleUITests;
+				PRODUCT_BUNDLE_IDENTIFIER = com.bootstraponline.AutoEarlGrey;
```

Comment:
This isn't a real project, it doesn't actually exist anywhere (other than as a fixture). I updated to replace `bootstraponline` with `Google`.